### ISSUE TITLE
feature request: add column "form name" to Procedures Pending report

### ISSUE THREAD

**Issue by htuck** [LibreHealthIO/LibreEHR#1040]
what it says: please consider adding to that report a column for the name of the form that needs to be signed.

**Comment by aethelwulffe** [comment]
should that be the name of the form, or should it be a Reason or Title as well for context?

**Comment by htuck** [comment]
I think of this column as displaying whatever form name shows in the encounter listing, per the pic.  As a guide for somebody using the report to determine which form has been left unsigned- if it's the vitals form, the chart auditor won't ask the MD to sign it; if it's a clinical form (ROS, PE, SOAP etc) the auditor knows the form's provider needs to be told about it; if it's a Order Form the transcriptionist will know they need to transcribe it.

![formname](https://user-images.githubusercontent.com/8339489/36802242-47e0e474-1c69-11e8-820f-d25be842a69d.png)

**Comment by teryhill** [comment]
The report will show the name of the ones not esigned. May need some tweaking.

![image](https://user-images.githubusercontent.com/4655366/36802810-eb2b0b68-1c83-11e8-85d3-7d5e45a06449.png)

**Comment by htuck** [comment]
That's perfect, @teryhill , thanks!  When will that mod be in the repo?  That's worth waiting for that part of the video for.

**Comment by teryhill** [comment]
It is there now @htuck  just check the box marked Details.

**Comment by htuck** [comment]
umm... duuh, I had been looking at the Reports/ Procedures/ Pending Res  when I suddenly remembered Reports/ Visits/ Encounters...
Thanks for the wakeup!  I'm going to make more coffee...

**Comment by aethelwulffe** [comment]
@htuck Sounds like something to *carefully* document in the instructions for the report use as well as for workflow documentation.
I suggest keeping this open until the current documentation project or video is complete.

**Comment by htuck** [comment]
Yes, @aethelwulffe at top of doc, in red: ENSURE ADEQUATE CAFFEINATION BEFORE ATTEMPTING
let's keep it open... 

And in the meantime I found another problem, which may be with the report or something else.
Pic in a bit; details first:  e-sign forms in the encounter and they still show as unsigned in the Encounters report
- this occurs when e-signed by both the original form owner or by another (e.g., transcriptionist); after simply refreshing the screen after the e-sig; after everybody logs out / back in and running the enc report again; after clearing the current patient, logging out/ back in as the original owner and e-signing again; many permutations of these steps
- occurs in my current git clone, release 2 rc1 and ehr.stable.librehealth.io/.

![procordesignotake](https://user-images.githubusercontent.com/8339489/36921318-c9c76dd6-1e18-11e8-97a1-d5f244e27280.png)
